ChangeLog.d/fix-declaration-of-mbedtls_ecdsa_sign_det_restartable-function.txt

```diff
@@ -0,0 +1,7 @@
+Bugfix
+   * Fix declaration of mbedtls_ecdsa_sign_det_restartable() function
+     in the ecdsa.h header file. The function was declared only when 
+     the macro MBEDTLS_ECDSA_SIGN_ALT is not defined and thus
+     producing a build warning when the macro MBEDTLS_ECDSA_SIGN_ALT
```

Comment:
I think that's enough. 
```suggestion
     in the ecdsa.h header file. There is a build warning when the macro 
     MBEDTLS_ECDSA_SIGN_ALT is defined.
```
Please fix me if my suggestion looks strange.

Reply:
The suggestion looks good and have applied it.